libraries/Filter/HarmonicNotchFilter.cpp

```diff
@@ -228,27 +317,16 @@ void HarmonicNotchFilter<T>::update(float center_freq_hz)
 
     _num_enabled_filters = 0;
     // update all of the filters using the new center frequency and existing A & Q
-    for (uint8_t i = 0; i < HNF_MAX_HARMONICS && _num_enabled_filters < _num_filters; i++) {
+    for (uint16_t i = 0; i < HNF_MAX_HARMONICS && _num_enabled_filters < _num_filters; i++) {
```

Comment:
_num_harmonics is uint8_t, probably both should be the same time

Reply:
changed to uint8_t